doc/next-steps/customizing-your-list.md

```diff
@@ -0,0 +1,314 @@
+import { hatchifyReact, HatchifyProvider, createJsonapiClient } from "@hatchifyjs/react"
+import { createTheme, ThemeProvider } from "@mui/material"
+import * as Schemas from "../schemas.js"
+
+export const hatchedReact = hatchifyReact(createJsonapiClient("/api", Schemas))
+
+const TodoList = hatchedReact.components.Todo.Collection
+const TodoEmpty = hatchedReact.components.Todo.Empty
+const TodoColumn = hatchedReact.components.Todo.Column // 👀
+
+const App: React.FC = () => {
+  return (
+    <ThemeProvider theme={createTheme()}>
+      <HatchifyProvider>
+        <TodoList>
+          <TodoEmpty>
+            <strong>There are no todos. Time to take a break!</strong>
+          </TodoEmpty>
+          {/* 👀 */}
+          <TodoColumn type="replace" field="name" label="ToDo" renderDataValue={({ value }) => <strong>{value}</strong>} renderHeaderValue={({ column: { label } }) => <strong>{label} Items</strong>} sortable={true} />
+        </TodoList>
+      </HatchifyProvider>
+    </ThemeProvider>
+  )
+}
+
+export default App
+```
+
+Your app should now look like this:
+
+![localhost_3000_ (2)](https://github.com/bitovi/hatchify/assets/60432429/72a0c73e-fc68-4fc2-b4a8-1f5ebc9fe44c)
+
+Notice how the column for the `name` field has been replaced with our custom column.
+
+#### How it works
+
+- Just like `Todo.Empty`, we need a reference to the `Todo.Column` component. The following aliases it to make it easier to reference:
+
+  ```tsx
+  const TodoColumn = hatchedReact.components.Todo.Column
+  ```
+
+- Setting the `type` prop to `replace` instructs Hatchify to render all the usual columns in our list, while _only_ applying our customizations to the column that corresponds to the value of the `field` prop on `TodoColumn`.
+
+  ```tsx
+  type = "replace"
+  field = "name"
+  ```
+
+- The `label` prop allows us to overwrite the column's default header with a custom string.
+
+  ```tsx
+  Label = "To Do"
+  ```
+
+- The `renderDataValue` prop allows us to pass in a callback that returns JSX. That JSX will fully overwrite the contents of each data cell.
+
+  ```tsx
+  renderDataValue={({ value }) => <strong>{value}</strong>}
+  ```
+
+  The callback accepts one argument: an object with a `value` property and a `record` property. `value` is equal to the value that would have otherwise been rendered in each data cell. `record` is equal to the full record represented by a given data cell's row.
+
+  - Alternatively, you could use the `DataValueComponent` prop instead of `renderDataValue`. It does the same thing, but instead of accepting a callback it accepts a React component that optionally accepts a `record` prop and a `value` prop. It would look something like this:
+
+    ```tsx
+    // Define your component:
+    const CustomComponent = ({ value }: { value: string }) => <strong>{value}</strong>
+    ```
+
+    ```tsx
+    // Inside your TodoColumn, replace the `renderHeaderValue` prop with:
+    DataValueComponent = { CustomComponent }
```

Comment:
spacing is a bit weird here, should just be `DataValueComponent={CustomComponent}`

Reply:
Unfortunately my auto-formatting keeps injecting the spaces--any tips on how to fix that? 🤔

Reply:
Good call Roye! Done